src/javascript/app/pages/user/metatrader/metatrader.ui.js

```diff
@@ -584,6 +588,66 @@ const MetaTraderUI = (() => {
         });
     };
 
+    const setDemoTopupStatus = (is_topped_up = false) => {
+        const el_demo_topup_btn  = getElementById('demo_topup_btn');
+        const el_demo_topup_info = el_demo_topup_btn.previousSibling;
+        const el_loading         = el_demo_topup_btn.parentElement.firstChild;
```

Comment:
for these tow elements (top up info and loading) can we fetch them in a way that's not so dependent on HTML structure in case elements move later?

Reply:
Noted and addressed.